src/main/java/org/kpmp/Query.java

```diff
@@ -161,4 +161,8 @@ public ParticipantSummaryDataset participantSummaryDataset(String redcap_id) thr
 			return emptyResult;
 		}
 	}
+
+	public ParticipantSummaryDataset participantClinicalDataset(String redcap_id) throws Exception {
```

Comment:
This is a little curious, and I thought this made the most sense. Other options included:
1. Duplicating all of the code inside of participantSummaryDataset (rather than calling it directly

2. Setting up another repository to call just the clinical data. This appeared to be overkill wrt duplicate code and testing.

3. Adjust the graphql query to send 'participantDataset', including in it both the summary and clinical datasets. This would be the least code, but removes some of the idea of separation of concerns on the backend. The front-end could in theory still have that separation in multiple function calls to the same endpoint (asking for different data), but I found this path to be a little confusing from the backend perspective.f